Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -28,7 +28,7 @@ public static float CalculateBmi(float weight, float height)
             else if (weight <=0)
             {
                 Console.WriteLine("Failed calculating BMI. Reason:");
-                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                Console.WriteLine("Weight cannot be less than zero, but was " + weight + ".");
```

Comment:
Take a look at this: [Readme.md of Homework/4](https://github.com/csinn/CSharp-From-Zero-To-Hero/blob/Chapter1/Homework/4/README.md)
Maybe this can help you understand what to print when.